### DISCUSSION TITLE
Disabling global formatting for custom script

### DISCUSSION THREAD

**Discussion by bohorok** [polybar/polybar#2944] [Q&A] [answered]
I've setting foreground in [bar/my bar] section. I would like this formatting not to format my module in [custom/script]. The script has its own colors set. how to configure polybar so that he does not overwrite the colors set in the script. Is there any solusion for this problem ?

**Comment by patrick96** [comment]
If a script sets its own foreground color using `%{F#...}`, that color is used and not the foreground color set in the bar section. How does polybar overwrite the colors in the script?

**Comment by bohorok** [comment]
Could you develop a little how to do it? .  How to use it ?
I expressed myself badly. I am looking for a solution to make the script overwrite Polybar formatting.

**Reply by patrick96** [reply]
You can use [formatting tags](https://github.com/polybar/polybar/wiki/Formatting#format-tags) in your script's output for that.

**Comment by bohorok** [comment]
It does not work. I have line in  config.ini file:
`label = %{F#f00}%output%{F-}%`
I see on the bar three dots in the selected color instead of the script output in that color.
do you know what I'm doing wrong. ?

**Reply by patrick96** [reply, marked as answer]
Do you have `label-maxlen` set? If not, then those dots are most likely the output of your script (at least the first line)

**Comment by bohorok** [comment]
Finally works. I combined with the script and works well. I was set 
`label-maxlen`. 
Your suggestion was right to increase the width of the module. I had to experiment because at the output in the bar I got the last piece of formatting code: `{F-}`.  The correct record turned out to be: 
 `label = %{F#f00}%output%`.  I don't know why there is such a provision in the official documentation: 
`label = %{F#f00}%output%{F-}%`
great thanks for guiding me.

**Comment by bohorok** [comment]
Thanks again
